src/SDPoptimize.jl

```diff
@@ -518,13 +518,13 @@ function sdp_forward_simulation(model::SPModel,
     nb_scenarios = size(scenarios)[2]
 
     costs = zeros(nb_scenarios)
-    states = zeros(TF,nb_scenarios)
-    controls = zeros(TF-1,nb_scenarios)
+    states = zeros(TF,nb_scenarios,model.dimStates)
+    controls = zeros(TF-1,nb_scenarios,model.dimControls)
 
 
```

Comment:
Could this be simplified with the use of sdp_forward_simulation instead of a loop over the scenarios ?